### ISSUE TITLE
FR: Disable IPv4 and operate as IPv6-only

### ISSUE THREAD

**Issue by buraglio** [tailscale/tailscale#13952]
### What are you trying to do?

Operate a TailNet as IPv6-only

### How should we solve this?

Given US Government requirements for both ZTA (which TailScale is a great compliment to) for removal of 
IPv4, it would be *very* advantageous to allow for the disablement of IPv4 across a given Tailnet. The Tailnet exit nodes could pretty easily provide NAT64 capabilities making this product a *very* desirable, easy win for hitting two major milestones: 
ZTA and IPv6-only. 

As mentioned in [13951](https://github.com/tailscale/tailscale/issues/13951), ULA is preferred below IPv6 in normal source address selection, so even with magic DNS AAAA records in a dual stacked environment there will be no notable IPv6 traffic in the presence of legacy IP.

### What is the impact of not solving this?

Since this is a "service" that would be purchased and not owned by the USG, there is no hard requirement for operating as IPv6-only, however, it would be a useful feature to allow for the disablement of the legacy IP protocol in favor of operating using only the current working standard for IP transport (IPv6). TS is a good contender for ZTA, and the removal of the complexity of running legacy IP would be a nice feature.

**Comment by shladek** [comment]
DisableIPv4 : true is an option that can be set in your ACL to make tailscale only use ipv6 on your tailnet. It's noted [here](https://tailscale.com/kb/1023/troubleshooting#cgnat-conflicts) . 

Is that what you mean?

**Comment by buraglio** [comment]
Yes, this is **_exactly_** what I was looking for, thank you for the pointer. Any chance this can be made a little more user visible?

**Comment by shladek** [comment]
It's also in the ACL syntax documentation [here](https://tailscale.com/kb/1337/acl-syntax#network-policy-options)
